.github/workflows/lint.yml

```
@@ -61,29 +41,6 @@ jobs:
         with:
           python-version: "3.8"
 
-      - name: Install Poetry
-        uses: snok/install-poetry@v1.1.0
-        with:
-          virtualenvs-create: true
-          virtualenvs-in-project: true
-
-      - name: Load cached venv
-        id: cached-poetry-dependencies
-        uses: actions/cache@v2
-        with:
-          path: .venv
-          key: venv-${{ runner.os }}-${{ hashFiles('**/poetry.lock') }}
-
-      - name: Install dependencies
-        run: poetry install
-        if: steps.cached-poetry-dependencies.outputs.cache-hit != 'true'
-
-      - name: Activate venv
-        run: |
-          echo "VIRTUAL_ENV=$GITHUB_WORKSPACE/.venv" >> $GITHUB_ENV
-          echo "PYTHONPATH=$GITHUB_WORKSPACE" >> $GITHUB_ENV
-          echo "$GITHUB_WORKSPACE/.venv/bin" >> $GITHUB_PATH
-
       - name: pylint Generate Report
         run: >
           pylint --exit-zero --load-plugins=pylint_json2html -f jsonextended project_name |
```

I think we need to swap out `project_name` for `cl` here.
```suggestion
          pylint --exit-zero --load-plugins=pylint_json2html -f jsonextended cl |
```